server/core/store.go

```diff
@@ -67,13 +67,12 @@ type StoreInfo struct {
 // NewStoreInfo creates StoreInfo with meta data.
 func NewStoreInfo(store *metapb.Store, opts ...StoreCreateOption) *StoreInfo {
 	storeInfo := &StoreInfo{
-		meta:           store,
-		storeStats:     newStoreStats(),
-		leaderWeight:   1.0,
-		regionWeight:   1.0,
-		limiter:        make(map[storelimit.Type]*storelimit.StoreLimit),
-		minResolvedTS:  0,
-		lastAwakenTime: time.Now(),
```

Comment:
Have u tested when the cluster restarts?

Reply:
Yep, when restarting PD, PD will do the checking to determine whether it should send AwakenRegions or not. And if there still existed slow stores, parsing from etcd, it will return slowStoreIDs in AwakenRegions to TiKV side.